### PR TITLE
feat: upgrade to version 1.x of the API

### PR DESCRIPTION
**Pull request by bcoe**
upgrade to version 1.x of the API.

_Note: I've opted to continue exposing the export v1beta1 for the time being, this stops us from breaking all of the sample tests._

CC: @DanSanche

### CODE REVIEW

File: synth.py [hunk 1]
```diff
@@ -38,6 +38,10 @@
         'cloud.google.comgrafeas.io',
         'grafeas.io')
 
+s.replace('src/v1/*.js',
```
Comment (JustinBeckwith) [inline synth.py]: Guessing this is a bug in the generator or the protos?  Please file a bug in `googleapis/gapic-generator`, and link to the issue in the synth file :)

File: synth.py [hunk 2]
```diff
@@ -23,11 +23,11 @@
 
 # Run the gapic generator
 gapic = gcp.GAPICGenerator()
-versions = ['v1beta1']
+versions = ['v1beta1', 'v1']
 for version in versions:
     library = gapic.node_library('containeranalysis', version,
             config_path=f"/google/devtools/containeranalysis/artman_containeranalysis_{version}.yaml")
-    s.copy(library, excludes=['package.json', 'README.md', 'src/index.js'])
+    s.copy(library, excludes=['package.json', 'README.md', 'src/index.js', 'test/gapic-v1beta1.js'])
```
Comment (JustinBeckwith) [inline synth.py]: Why are we excluding the v1beta1 tests?

**Comment by JustinBeckwith** [comment]
@bcoe Could I trouble you to write the text of the breaking change in the PR description?  We probably want to be explicit to customers on what's changing here.

File: synth.py (hunk 2 above)
Comment (bcoe) [reply synth.py]: The tests import the client incorrectly; seeing as all the sample tests fail without this old client (it seems as though the new API surface is quite different) I'm going to instead add the old export.

File: synth.py (hunk 1 above)
Comment (bcoe) [reply synth.py]: we've patched this internally now :+1:

**Pull request merged**